src/Tasks/Microsoft.Build.Tasks.csproj

```diff
@@ -999,9 +999,4 @@
     -->
     <Content Update="@(Content)" Pack="false" />
   </ItemGroup>
-
-  <Target Name="CopyMscorlib" AfterTargets="Build">
```

Comment:
@jeffkl Is it OK to remove these? Does RoslynCodeTaskFactory need them or something?

Reply:
Yes we need them so that RoslynCodeTaskFactory can compile assemblies.  They're just ref assemblies